README.md

```diff
@@ -28,11 +28,15 @@ A **simple**, **powerful** and **flexible** Commerce Seller Application built on
 * Create, read, update, delete user groups and users
 * Read, Update Orders
 
-## Requirement
-Create an OrderCloud Marketplace instance (https://portal.ordercloud.io)
+## Requirements
+An OrderCloud Marketplace instance (https://portal.ordercloud.io) with the following (minimum):
+  - An admin api client *without* client secret defined
+  - An admin user with the [correct permissions](https://github.com/Sitecore/Sitecore.Commerce.Headstart.ReactAdmin/blob/main/src/constants/app-permissions.config.ts#L12-L20) (or FullAccess)
+ 
+ Don't have a marketplace? Check out the [section on seeding](#seeding-a-new-marketplace) to create a new marketplace based on the play shop product set
 
 ## How do I get started? 
-Using the Deploy Button below, you'll deploy on Vercel the Next.js project as well as connect it to your Sitecore Commerce OrderCloud sandbox.
+Using the Deploy Button below, you'll deploy on Vercel the Next.js project as well as connect it to your OrderCloud Marketplace.
```

Comment:
Removed Sitecore Commerce because I don't think we want to confuse people with thinking this has anything to do with XC

And removed sandbox, because this is not specific to sandbox environment